docs/stdlib/set.rst

```diff
@@ -108,9 +108,9 @@ Sets
 
     :index: intersection
 
-    Test the membership of an element in a set.
+    Performs a logical check of membership for an element in a set.
 
-    Set membership operators ``in`` and ``not in`` that test for each
+    Set membership operators ``in`` and ``not in`` test for each
     element of ``A`` whether it is present in ``B``.
```

Comment:
```suggestion
    Set membership operators ``in`` and ``not in`` test whether each element of the
    left operand is present in the right operand.
```

Since we haven't yet referred to the operands as `A` and `B`, this rewrite simply refers to them as what they are.